.circleci/config.yml

```diff
@@ -1,408 +1,46 @@
-        type: env_var_name
-      use-docker-credentials-store:
-        type: boolean
-    executor: <<parameters.executor>>
-
-    steps:
-      - docker/check:
-          docker-username: <<parameters.docker-username>>
-          docker-password: <<parameters.docker-password>>
-          use-docker-credentials-store: <<parameters.use-docker-credentials-store>>
-
-  test-credentials-store:
-    parameters:
-      executor:
-        type: executor
-      helper-name:
-        type: enum
-        enum: ["", "pass", "osxkeychain"]
-        default: ""
-      docker-username:
-        type: env_var_name
-      docker-password:
-        type: env_var_name
-    executor: <<parameters.executor>>
-
-    steps:
-      - docker/install-docker-credential-helper:
-          helper-name: <<parameters.helper-name>>
-      - docker/configure-docker-credentials-store:
-          helper-name: <<parameters.helper-name>>
-      - run:
-          name: Test login
-          command: |
-            LOGIN_RESULT=$(echo "$<<parameters.docker-password>>" \
-            | docker login -u "$<<parameters.docker-username>>" --password-stdin)
-            set -x
-            echo "$LOGIN_RESULT" | grep "Login Succeeded"
-            set +x
-            if echo "$LOGIN_RESULT" | grep -q "WARNING"; then
-            echo "Error - warning found in login result"
-            exit 1
-            fi
-
-  test-create-workspace:
-    docker:
-      - image: cimg/base:stable
-    steps:
-      - run:
-          name: Description
-          command: |
-            echo "This step simply creates a file and saves it to a workspace. This tests the attach_at parameter in the build command."
-            echo "true" > verify.txt
-      - persist_to_workspace:
-          root: ./
-          paths:
-            - verify.txt
-
-  test-build-command-workspace:
-    docker:
-      - image: cimg/base:stable
-    steps:
-      - checkout
-      - setup_remote_docker
-      - docker/build:
-          dockerfile: test.Dockerfile
-          image: cpeorbtesting/docker-orb-test
-          tag: $CIRCLE_BUILD_NUM-$CIRCLE_SHA1
-          attach-at: "./"
-      - run:
-          name: Check for workspace
-          command: |
-            echo "Ensuring workspace was attached."
-            if test -f "./verify.txt"; then
-              echo "Success"
-            else
-              echo "Error"
-              pwd
-              ls
-              exit 1
-            fi
+  orb-tools: circleci/orb-tools@11.1
+  shellcheck: circleci/shellcheck@3.1
 
 workflows:
-  lint_pack-validate_publish-dev:
-    unless: << pipeline.parameters.run-integration-tests >>
+  lint-pack:
     jobs:
-      - orb-tools/lint
+      - orb-tools/lint:
+          filters:
+            tags:
+              only: /.*/
       - orb-tools/pack:
-          requires: [orb-tools/lint]
-      - orb-tools/publish-dev:
+          filters:
+            tags:
+              only: /.*/
+      - orb-tools/review:
+          exclude: RC009
```

Comment:
Why excluding RC009? Scripts should be migrated to the scripts directory.

Reply:
To keep the PR focused on the migration. I'm gonna follow up with a few PRs fixing bugs and moving shell to the scripts files.